code/modules/urist/modules/shipbattles/overmapships/components/components.dm

```diff
@@ -5,15 +5,17 @@
 	var/health = 100
 	var/mob/living/simple_animal/hostile/overmapship/mastership = null
 	var/broken = FALSE
-	var/targeted = TRUE
+	var/targeted = FALSE
 	var/last_activation = null
 
 /datum/shipcomponents/proc/BlowUp()
 //	qdel(src)
 	mastership.health -= 100
 	broken = TRUE
+	targeted = FALSE
 	name = "destroyed [initial(name)]"
 	if(mastership.health <= 0)
+		mastership.health = 0
```

Comment:
Is this necessary?

Reply:
Whoops! This was an artefact left from previous attempts to get the nanoUI to not display negative pirate health on destruction. This was fixed further down the line and I forgot to remove this.
Patched in d5c3d44